storehaus-mysql/src/main/scala/com/twitter/storehaus/mysql/MySqlStore.scala

```diff
@@ -97,59 +100,68 @@ class MySqlStore(protected [mysql] val client: Client, table: String, kCol: Stri
   protected [mysql] def commitTransaction : Future[Unit] = client.query(COMMIT_TXN_SQL).unit
   protected [mysql] def rollbackTransaction : Future[Unit] = client.query(ROLLBACK_TXN_SQL).unit
 
-  protected [mysql] def executeMultiInsert[K1 <: MySqlValue](kvs: Map[K1, MySqlValue]) = {
-    val insertSql = MULTI_INSERT_SQL_PREFIX + Stream.continually("(?, ?)").take(kvs.size).mkString(",")
-    val insertParams = kvs.map { kv =>
-      List(MySqlStringInjection(kv._1).getBytes, MySqlStringInjection(kv._2).getBytes)
+  protected [mysql] def executeMultiInsert[K1 <: MySqlValue](
+      kvs: Map[K1, MySqlValue]): Future[Result] = {
+    val insertSql = MULTI_INSERT_SQL_PREFIX +
+      Stream.continually("(?, ?)").take(kvs.size).mkString(",")
```

Comment:
can we say `Iterator.continually` which I think is slightly faster than a Stream if we are just calling `mkString`?